test/ecstresstest.c

```diff
@@ -18,7 +18,7 @@
 
 #define NUM_REPEATS "1000000"
 
-static ossl_intmax_t num_repeats;
+static intmax_t num_repeats;
```

Comment:
gotta use `intmax_t` to be sure that it doesn't overflow on 1M repeats or whatever number the user passed to `-num` since surely they want to stress test for weeks, months or even years, but good thing it is signed! :)